### PR TITLE
fix: REPL comment for pull request

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

Hopefully it will work. :)

### CODE REVIEW

File: .github/workflows/repl.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@ jobs:
           github-token: ${{ secrets.BOT_TOKEN }}
           script: |
             const { default: repl } = await import('${{ github.workspace }}/scripts/actions/repl.js');
-            await repl();
+            await repl(github, context);
```
Comment (nicolo-ribaudo) [inline .github/workflows/repl.yml:23]: These are not JS variables -- could you move the action to https://github.com/babel/actions and follow how, for example, https://github.com/babel/actions/blob/89fc1d36b649d3b6f2ba40e1051555c999d69842/publish-github-release/main.js#L5 gets these values?

Comment (liuxingbaoyu) [reply .github/workflows/repl.yml:23]: They are defined by `actions/github-script`, but if you feel better moving to https://github.com/babel/actions I can do that too.

**Review by nicolo-ribaudo — APPROVED** [review]
Let's try to get CI green for now, we can move it to the actions repo later.

**Comment by liuxingbaoyu** [comment]
Unfortunately it still fails, I will investigate. :)

**Comment by liuxingbaoyu** [comment]
This succeeded at https://github.com/babel/babel/pull/16187#issuecomment-1866219020, it looks like it's just failing on the branch in the `babel/babel` repository.
https://circleci.com/api/v2/project/github/babel/babel/pipeline?branch=pull/16252
vs
https://circleci.com/api/v2/project/github/babel/babel/pipeline?branch=pull/16187

**Comment by nicolo-ribaudo** [comment]
Feel free to create a branch in this repo to test :)

**Pull request merged**